modules/nf-core/proteus/readproteingroups/tests/main.nf.test

```diff
@@ -0,0 +1,79 @@
+nextflow_process {
+
+    name "Test Process PROTEUS_READPROTEINGROUPS"
+    script "../main.nf"
+    process "PROTEUS_READPROTEINGROUPS"
+    config "./nextflow.config"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "proteus_readproteingroups"
+    tag "proteus"
+    tag "proteus/readproteingroups"
+
+    test("test_proteus_readproteingroups") {
+
+        when {
+            params{
+                module_args = "--contrast_variable fakeBatch --sample_id_col Experiment  --measure_col_prefix \"LFQ intensity\" --round_digits 3 --norm_function \"normalizeQuantiles\""
```

Comment:
I normally just write tags directly in the config of the module and I do not use params, but I do not know if there is a standard regarding this.